report/renderer/topbar-features.js

```diff
@@ -270,7 +270,8 @@ export class TopbarFeatures {
       this.highlightEl = this._dom.createChildOf(this.stickyHeaderEl, 'div', 'lh-highlighter');
 
       // Update sticky header visibility and highlight when page scrolls/resizes.
-      const scrollParent = this._getScrollParent(this._dom.rootEl);
+      const scrollParent = this._getScrollParent(
+        this._dom.find('.lh-container', this._dom.rootEl));
```

Comment:
FWIW `this._getScrollParent(this.stickyHeaderEl)` might be more robust against future changes to the scrollable container. Up to you though.

Reply:
I've already verified the previous method still works and would prefer to go with that to continue the release today. Am open to someone/you giving this more consideration later.